rust/blockstore/src/arrow/ordered_blockfile_writer.rs

```diff
@@ -133,6 +144,25 @@ impl ArrowOrderedBlockfileWriter {
 
         Self::complete_current_delta::<K, V>(&mut inner);
 
+        let num_block_cache_hits = self
+            .num_block_cache_hits
+            .load(std::sync::atomic::Ordering::Relaxed);
+        let num_block_cache_misses = self
```

Comment:
Foyer already exposes the underlying hit rate and miss rate - i think this is redundant?

https://github.com/chroma-core/chroma/blob/8f7a0ff7b503a8528d0de1d35d694118ae2e105e/rust/cache/src/foyer.rs#L251

Reply:
Also if we are doing it - i i think it should be a metric not a trace

Reply:
ah yep missed that
I think it's actually kinda useful having it on the trace; if all you have is a metric point with a low cache hit rate it could be pretty hard to debug
will leave out for now

Reply:
ah makes sense